From fa3e37ff7e39507703f2ab12d8f606f47ddcc32d Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Tue, 31 Oct 2023 10:52:03 +0100
Subject: [PATCH 0044/1695] plat-stm32mp1: conf: fix order for
 CFG_REGULATOR_FIXED

Changes CFG_REGULATOR_FIXED config setting location to match alphabetical
order.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git eee4c0d256830ef739cc48ab70bb41be2ef49f61 plat-stm32mp1: conf: fix order for CFG_REGULATOR_FIXED]
Change-Id: I913762b26d8e819375a3b5dcbc2ebb95d9be1eea
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne-cherry-picked from b2c13caa750617f7037710b81d69db266659224b]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 90f6c853b..43dfa9b71 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -99,10 +99,10 @@ $(call force,CFG_DRIVERS_REGULATOR,y)
 $(call force,CFG_GIC,y)
 $(call force,CFG_INIT_CNTVOFF,y)
 $(call force,CFG_PSCI_ARM32,y)
+$(call force,CFG_REGULATOR_FIXED,y)
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 $(call force,CFG_SM_PLATFORM_HANDLER,y)
 $(call force,CFG_STM32_SHARED_IO,y)
-$(call force,CFG_REGULATOR_FIXED,y)
 
 ifeq ($(CFG_STM32MP13),y)
 $(call force,CFG_BOOT_SECONDARY_REQUEST,n)
-- 
2.47.2

